### PR TITLE
Skip stats filter test when operator is not used

### PR DESCRIPTION
**Pull request by bianpengyuan**

### CODE REVIEW

File: tests/integration/telemetry/stats/prometheus/stats_filter_test.go [hunk 1]
```diff
@@ -106,6 +107,11 @@ func buildQuery() (sourceQuery, destinationQuery string) {
 // proxy bootstrap config. To avoid flake, it does not verify correctness of metrics, which
 // should be covered by integration test in proxy repo.
 func TestStatsFilter(t *testing.T) {
+	if !usingOperator {
+		// TODO(bianpengyuan): remove this condition when operator is used in all tests.
```
Comment (mandarjog) [inline tests/integration/telemetry/stats/prometheus/stats_filter_test.go]: Can you instead do t.Skip with an attached issue?

Comment (bianpengyuan) [reply tests/integration/telemetry/stats/prometheus/stats_filter_test.go]: Switch to use t.Skip. This is not an issue. This test just should not run when using helm installation, which does not have option for stats filter. This condition could be removed when helm installation is removed completely from CI.

**Comment by gargnupur** [comment]
@bianpengyuan : should we do this for StatcTcpFilter test too?

**Comment by howardjohn** [comment]
We only support operator now in tests, I think this is no longer needed

**Pull request closed without merging**